server/src/test/scala/org/scassandra/server/e2e/PreparedStatementPreparationVerificationTest.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2014 Christopher Batey and Dogan Narinc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.scassandra.server.e2e
+
+import dispatch.Defaults._
+import dispatch._
+import org.scalatest.concurrent.ScalaFutures
+import org.scassandra.server.AbstractIntegrationTest
+import org.scassandra.server.priming.{PrimingJsonImplicits, PreparedStatementPreparation}
+import spray.json._
+
+class PreparedStatementPreparationVerificationTest extends AbstractIntegrationTest with ScalaFutures {
+
+  import PrimingJsonImplicits._
+
+  val primePreparedSinglePath = "http://localhost:8043/prepared-statement-preparation"
```

Comment:
I need to remove this hard coding of host/port but don't worry about it we'll raise a separate issue to get rid of the duplication for all e2e tests in the server